pkgs/dart_services/test/flutter_web_test.dart

```diff
@@ -94,23 +91,3 @@ void defineTests() {
     });
   });
 }
-
-class _FakeImportDirective implements ImportDirective {
```

Comment:
👍 - very nice; when I'd looked at doing this previously I got hung up on not being able to mock these anymore

Reply:
I did as well, but decided it wasn't worth matching exactly.